src/ui/reportView/callReport.ts

```diff
@@ -96,7 +98,7 @@ function createCommand(commandURI: string, harnessFile: string, harnessName: str
 }
 
 // 	Find the path of the report from the harness name
-function findPath(dir: string, filename: string): string {
+async function findPath(dir: string, filename: string): Promise<string> {
```

Comment:
why has this been changed to `async`?

Reply:
I wanted the function to be blocking, without verifying that the path exists in the target directory, i do not want the link to view the file to be created and presented to the user. There are issues where the underlying project that creates the report or kani can be slow with generating the report and in that case, I think it would create for a better UX to not create a link than to create a broken link.